src/webserver/src/php_amule_lib_standalone.cpp

```diff
@@ -296,21 +294,18 @@ void php_native_search_start_cmd(PHP_VALUE_NODE *)
 		return;
 	}
 	cast_value_dnum(&si->var->value);
-	int avail = si->var->value.int_val;
 
 	if ( !(si = get_scope_item(g_current_scope, "__param_5")) ) {
 		php_report_error(PHP_ERROR, "Invalid or missing argument 6 (min size)");
 		return;
 	}
 	cast_value_dnum(&si->var->value);
-	int min_size = si->var->value.int_val;
 
 	if ( !(si = get_scope_item(g_current_scope, "__param_6")) ) {
 		php_report_error(PHP_ERROR, "Invalid or missing argument 7 (max size)");
 		return;
 	}
 	cast_value_dnum(&si->var->value);
-	int max_size = si->var->value.int_val;
 
 	printf("php_native_search_start_cmd: search=%s \n", search);
```

Comment:
You should instead expand this `printf` to display the values of the other parameters, too. These functions are used to test the interpreter and aMule-specific functions, thus it's better to display the interpreted values than to silently ignore them.

Reply:
ok, done